packages/data-service-generator/src/prepare-context.ts

```diff
@@ -126,6 +129,16 @@ export function prepareEntityPluralName(entities: Entity[]): Entity[] {
   return currentEntities;
 }
 
+export function isGrpcGenerate(
```

Comment:
Minor comment, but can we rename this method (and all usages) to "shouldGenerateGrpc()" ?